README.md

```diff
@@ -86,105 +176,156 @@ Some examples are given below:
+    | 6      | polygons | select | NA  | NA       | polygonID:fileLocation;organizationID;conatctID | NA    |
+
 Notes:
 
--   In examples 2 and 4 where multiple columns and tables were selected respectively, a `;` was used to separate the values. Throughout this entire document when multiple values need to listed in a single cell, the `;` symbol should be used to separate discrete values.
+-   In examples 2 and 4 where multiple columns and tables were selected
+    respectively, a `;` was used to separate the values. Throughout this entire
+    document when multiple values need to listed in a single cell, the `;`
+    symbol should be used to separate discrete values.
 
--   In examples 3 and 5 where all the columns in a table and all the tables were selected respectively, the keyword `all` was used. Similar to the `;` symbol, the keyword `all` may be used in a cell to mean everything.
+-   In examples 3 and 5 where all the columns in a table and all the tables
+    were selected respectively, the keyword `all` was used. Similar to the `;`
+    symbol, the keyword `all` may be used in a cell to mean everything.
 
--   In example 6, all columns between 'polygonID' and 'fileLocation' were selected (inclusively), by separating them with a ":". The ";" symbol was also used to specify two more additional columns for inclusion.
+-   In example 6, all columns between 'polygonID' and 'fileLocation' were
+    selected (inclusively), by separating them with a ":". The ";" symbol was
+    also used to specify two more additional columns for inclusion.
 
--   The **ruleId** column is mandatory for all rules and each value is unique across the entire sheet (`sharing.csv`). While the recommended value is a number, the `ruleId` column can take either a number or a string as a value. If it is a string, it should not have any spaces in it. The recommended standard is to use [snake_case](https://en.wikipedia.org/wiki/Snake_case) for values in this column. See example 3 for an illustration of this case.
+-   The **ruleId** column is mandatory for all rules and each value is unique
+    across the entire sheet (`sharing.csv`). It must be a number.
 
 #### 3.2. Filtering Rows
 
-Once the columns and tables for inclusion have been specified, users can specify which rows should be shared using rules with the `filter` mode. Note that rules that filter can use values in any columns, including columns that are not being shared in the final output. To specify a `filter` rule, users need to specify the table or tables in the `table` column, and define the `mode` as filter. Then users can specify the columns which the filter will act on in the `key` column, specify the nature of the filter using the `operator` column, and the filter values in the `value` column. The general structure for the filter argument is:
-
-```         
-  **column name** **operator** **value**
```

Comment:
This isn't rendering or displaying on my build.

Reply:
good catch. fixed now.